frappe/translations/de.csv

```diff
@@ -1428,6 +1428,7 @@ Landing Page,Zielseite,
 Language,Sprache,
 Language Code,Sprachcode,
 "Language, Date and Time settings","Einstellungen zu Sprache, Datum und Uhrzeit",
+Last 6 months,Vergangene zwei Quartale,
```

Comment:
Actually, your translation is even more correct than the original! 😅
“Last 6 months” in fact doesn’t properly describe the last two quarters.
Also, I think we should add “Last half-year” to get_timespan_date_range() which isn’t the same as the last two quarters, but is widely used in business.

Reply:
Yeah, I actually started implementing this but decided it was too much of a distraction from what I wanted to get done 😅